src/components/header/header-component.vue

```diff
@@ -8,13 +8,13 @@
       <router-link class="header--nav-logo" to="/">
         <img src="../../images/dsw_logo.svg" alt="Denver startup week logo">
       </router-link>
-      <router-link class="header--nav-rewards-icon"  v-if="!['about', 'rewards'].includes($route.name)" to="/rewards">
+      <router-link class="header--nav-rewards-icon"  v-if="!['about', 'rewards', 'venue'].includes($route.name)" to="/rewards">
```

Comment:
You could change this array to a variable, or a function

variable approach:
```
const routes = ['about', 'rewards', 'venue']
// then your `v-if` looks like
... v-if="!routes.includes($route.name)"...
```

function approach:
```
routeExists = () => {
  const routes = ['about', 'rewards', 'venue']
  return routes.includes(this.$route.name)
}

// Then your `v-if` looks like
... v-if="!routeExists()" ...
```

Your naming might have to change if I'm reading this wrong but its the same concept

Reply:
Nice! I like it. Super clean!